### DISCUSSION TITLE
[Theseus/Desktop App] Desktop shortcuts for instances

### DISCUSSION THREAD

**Discussion by DJMoffinz** [modrinth/.github#230] [Feature requests]
I think it would be great to have some command line parameter you can use to launch a specific instance without going into the app, and be able to add this command as a desktop shortcut. a lot of launchers, including the official minecraft launcher lack this feature for some reason, and i think it would put you ahead of the competition as well as be very convenient.

thank you for your consideration.

**Comment by TymqTPL** [comment]
That would be very useful and cool, I think that would be very nice if this would be added

**Comment by noahwaseaten** [comment]
^^^

**Comment by zoenetic** [comment]
Would make using Modrinth on Steam Deck basically perfect!

**Comment by Bagaco3763** [comment]
please add this

**Comment by iam-best** [comment]
Modrinth Support confirmed that this will be added soon!
